matchms/filtering/add_parent_mass.py

```diff
@@ -17,7 +18,9 @@ def add_parent_mass(spectrum_in: SpectrumType) -> SpectrumType:
         try:
             charge = spectrum.get("charge")
             protons_mass = PROTON_MASS * charge
-            precursor_mz = spectrum.get("pepmass")[0]
+            precursor_mz = spectrum.get("precursor_mz", None)
```

Comment:
Nice, it now behaves as the reverse of the `add_precursor_mz()` filter.

Reply:
Thanks!